.circleci/config.yml

```diff
@@ -327,7 +330,7 @@ workflows:
   cut-release:
     triggers:
       - schedule:
-          cron: '0 19 * * 1'
+          cron: '0 2 * * 2'
```

Comment:
I'm updating this to run on Monday's at 7pm PST. CircleCI only supports UTC (https://support.circleci.com/hc/en-us/articles/115015481128-Scheduling-jobs-cron-for-builds-)